pkg/operator/ceph/disruption/clusterdisruption/add.go

```diff
@@ -72,12 +72,6 @@ func Add(mgr manager.Manager, context *controllerconfig.Context) error {
 		}),
 	}
 
-	// Watch for CephClusters
-	err = c.Watch(&source.Kind{Type: &cephv1.CephCluster{}}, &handler.EnqueueRequestForObject{})
```

Comment:
If we're removing the watch, what triggers the reconcile initially? The first reconcile happens automatically when the operator starts?